testing/test-helper.js

```diff
@@ -210,9 +210,9 @@ export class RequestBank {
 }
 
 export class BrowserController {
-  constructor(win) {
+  constructor(win, opt_doc) {
```

Comment:
A bit misleading. This is actually `ShadowRoot` or at least `Document|ShadowRoot`, right? Maybe then we should call it `rootNode`?